apps/frontend/readme.md

```diff
@@ -3,9 +3,68 @@
 ### Requires node 17
 
 ### Install deps
+
 `yarn install`
 
+### Login with Google Cloud with @nav.no user
+
+Do this in Google Chrome as it doesn't work in Firefox on Mac
+
+`gcloud auth login`
+
+Remember to flush sockets when you can't login
+
+`chrome://net-internals/#sockets`
+
 ### Run local, with port forward to dev-gcp
+
 `kubectl port-forward deployment/etterlevelse-backend`
 
-`yarn run start`
\ No newline at end of file
+`yarn run start`
+
+## Other
+
+### To use Yarn you need to
+
+1. `brew install yarn`
+2. `brew install node`
+
+## Common errors
+
+###
+
+#### Kubernetes error
+
+```
+frontend git:(master) ✗ kubectl port-forward deployment/etterlevelse-backend
+error: TYPE/NAME and list of ports are required for port-forward
+See 'kubectl port-forward -h' for help and examples
+```
+
+1. You need to add config (see below) in `.zshrc`.
+2. Run `gcloud auth login`
+3. Run `kubectl port-forward deployment/etterlevelse-backend 8080 --namespace teamdatajegerne`
```

Comment:
Assuming that the section below describes a set of functions in `.zshrc`, we might as well run `kpfe` instead of `kubectl port-forward deployment/etterlevelse-backend 8080 --namespace teamdatajegerne`. Either way works though, it's just that step 1 seems redundant if we're not going to use the short-name functions that we defined.

Reply:
I'm happy to approve it though, so you can choose what you want to do w/ that line :relieved: